build.sbt

```diff
@@ -119,7 +119,6 @@ lazy val coreTests = crossProject(JSPlatform, JVMPlatform)
   .settings(skip in publish := true)
   .settings(Compile / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat)
   .enablePlugins(BuildInfoPlugin)
-  .settings(scalacOptions ++= { if (isDotty.value) Seq() else Seq("-P:silencer:globalFilters=assert") })
```

Comment:
What is the rational behind this removal?

Reply:
I added this when we implemented the curried assert because otherwise you would get thousands of deprecation warnings for all the uses of of the old form of assert in our tests and the build would fail with fatal warnings enabled. Now that we have all the tests migrated we want those deprecation warnings so usage of the old form doesn't pass CI.